<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHM</class>
 <widget class="QWidget" name="IHM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IHM</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="technicien_2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mesures">
      <attribute name="title">
       <string>Mesures</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="Temperature">
           <property name="text">
            <string>Température</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="affichageTemperature"/>
         </item>
         <item>
          <widget class="QLabel" name="uniteTemperature">
           <property name="text">
            <string>°C</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="temperatureRessentie">
           <property name="text">
            <string>Température ressentie</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="affichageTemperatureRessentie"/>
         </item>
         <item>
          <widget class="QLabel" name="uniteTemperatureRessentie">
           <property name="text">
            <string>°C</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="humidite">
           <property name="text">
            <string>Humidité</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="affichageHumidite"/>
         </item>
         <item>
          <widget class="QLabel" name="uniteHumidite">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="luminosite">
           <property name="text">
            <string>Luminosité</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="affichageLuminosite"/>
         </item>
         <item>
          <widget class="QLabel" name="uniteLuminosite">
           <property name="text">
            <string>lux</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="pression">
           <property name="text">
            <string>Pression</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="affichagePression"/>
         </item>
         <item>
          <widget class="QLabel" name="unitePression">
           <property name="text">
            <string>hPa</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="altitude">
           <property name="text">
            <string>Altitude</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="affichageAltitude"/>
         </item>
         <item>
          <widget class="QLabel" name="uniteAltitude">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="technicien">
      <attribute name="title">
       <string>Mode Technicien</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
